### DISCUSSION TITLE
docker run failure

### DISCUSSION THREAD

**Discussion by zxzghr** [open-webui/open-webui#2495] [General]
docker run command:

docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://192.168.63.44:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.nju.edu.cn/open-webui/open-webui:main

docker ps result:
f7d160cb06a3        ghcr.nju.edu.cn/open-webui/open-webui:main   "bash start.sh"          28 minutes ago      Up 2 seconds (health: starting)   0.0.0.0:3000->8080/tcp   open-webui

docker logs 

Loading WEBUI_SECRET_KEY from file, not provided as an environment variable.
Loading WEBUI_SECRET_KEY from .webui_secret_key
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max
OpenBLAS blas_thread_init: pthread_create failed for thread 2 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max
OpenBLAS blas_thread_init: pthread_create failed for thread 3 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max
Failed to send telemetry event ClientStartEvent: can't start new thread
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max
OpenBLAS blas_thread_init: pthread_create failed for thread 2 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max
OpenBLAS blas_thread_init: pthread_create failed for thread 3 of 4: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max

Aborted!

How to fix it

**Comment by tengattack** [comment]
Try run docker with `--security-opt seccomp:unconfined`

**Reply by zhkm4321** [reply]
thx，it works

**Comment by ningmingxiao** [comment]
try this pr https://github.com/moby/moby/commit/9f6b562dd12ef7b1f9e2f8e6f2ab6477790a6594#diff-c079988079dadae9b6c26d5abbf7ade331a050ae29175cc073a154d8954e48dfR546   old docker doesn't support clone3.
